tests/client/amo/components/TestCategeories.js

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { findDOMNode } from 'react-dom';
+import {
+  renderIntoDocument,
+  findRenderedComponentWithType,
+} from 'react-addons-test-utils';
+import { Provider } from 'react-redux';
+
+import createStore from 'amo/store';
+import Categories from 'amo/components/Categories';
+import { getFakeI18nInst } from 'tests/client/helpers';
+
+
+const categories = [
+  {
+    application: 'firefox',
+    name: 'motorbikes',
+    slug: 'motorbikes',
+    type: 'extension',
+  },
+  {
+    application: 'android',
+    name: 'Travel',
+    slug: 'travel',
+    type: 'extension',
+  },
+  {
+    application: 'android',
+    name: 'Games',
+    slug: 'Games',
+    type: 'extension',
+  },
+  {
+    application: 'android',
+    name: 'Cats',
+    slug: 'cats',
+    type: 'theme',
+  },
+];
+
+describe('Categories', () => {
+  function render({ ...props }) {
+    const baseProps = {
+      clientApp: 'android',
+      categories,
+    };
+    const initialState = {
+      api: { clientApp: 'android', lang: 'fr' },
+      categories: { ...categories },
+    };
+
+    return findDOMNode(findRenderedComponentWithType(renderIntoDocument(
+      <Provider store={createStore(initialState)}>
```

Comment:
It's because we're using our internal `<Link>` component in that component. 😞

Would be removed once #1316 is fixed.